src/app/table/simple/terra-simple-table.component.ts

```diff
@@ -12,39 +12,59 @@ import { TerraSimpleTableRowInterface } from './row/terra-simple-table-row.inter
            })
 export class TerraSimpleTableComponent
 {
-    private _headerList:Array<TerraSimpleTableHeaderCellInterface>;
-    private _rowList:Array<TerraSimpleTableRowInterface>;
+    @Input() inputHeaderList:Array<TerraSimpleTableHeaderCellInterface>;
+    @Input() inputRowList:Array<TerraSimpleTableRowInterface>;
     @Input() inputUseHighlighting:boolean = false;
     @Input() inputIsStriped:boolean = false;
     
     constructor()
     {
     }
     
+    /**
+     * @deprecated
+     *
+     * @returns {Array<TerraSimpleTableHeaderCellInterface>}
+     */
     public get headerList():Array<TerraSimpleTableHeaderCellInterface>
     {
-        return this._headerList;
+        return this.inputHeaderList;
     }
     
+    /**
+     * @deprecated
+     *
+     * @param value
+     */
     public set headerList(value:Array<TerraSimpleTableHeaderCellInterface>)
     {
-        this._headerList = value;
+        this.inputHeaderList = value;
     }
     
+    /**
+     * @deprecated
+     *
+     * @returns {Array<TerraSimpleTableRowInterface>}
+     */
     public get rowList():Array<TerraSimpleTableRowInterface>
     {
-        return this._rowList;
+        return this.inputRowList;
     }
     
     public set rowList(value:Array<TerraSimpleTableRowInterface>)
     {
-        this._rowList = value;
+        this.inputRowList = value;
     }
     
+    /**
+     * @deprecated
+     *
+     * @param rowToDelete
+     */
     public deleteRow(rowToDelete:TerraSimpleTableRowInterface):void
     {
-        let index = this.rowList.indexOf(rowToDelete);
+        let index = this.inputRowList.indexOf(rowToDelete);
```

Comment:
das mit dem index... eigentlich genauso, man könnte aber gucken ob index == -1 (dann ist row nicht in der liste) aber das sollte, wie oben auch schon, nicht passieren